.nvmrc

```diff
@@ -1 +1 @@
-22.18.0
+22.20.0
```

Comment:
Just curious, why this minor version and not the [latest minor](https://nodejs.org/en/download/archive/v22.21.1) for node 22?

Reply:
That makes total sense, thanks for the reply!